### ISSUE TITLE
How can I make gradient flow through QuantTensor.

### ISSUE THREAD

**Issue by ppccww0201** [Xilinx/brevitas#631]
Hi, I'm simulating my network using Brevitas for HW implementation.

For some reasons, I have to genereate QuantTensor enveloping manipulated tensor `y` like this.
```python
y_q5 = QuantTensor(y.clone().detach().cuda(),
                   scale=torch.Tensor([re_scale]).clone().detach().cuda(),
                   zero_point=torch.Tensor([0]).clone().detach().cuda(),
                   bit_width=torch.Tensor([self.backbone_act_bitwidth]).clone().detach().cuda(),
                   signed=True,
                   training=self.training)
``` 

Of course, `y` is quantized by `re_scale`.
I implement `y_q5` in my network, and my network forwards well.
But back propagation path is broken at `y_q5`.
So the layers before `y_q5` cannot be learned.

How can I make gradient flow through `y_q5`?

Thank you!!

p.s. The value of `y` should not be changed. So I can't use QuantIdentity.

**Comment by ppccww0201** [comment]
I think `.clone().detach()` is the problem.

Resolved!

**Comment by ppccww0201** [comment]
I think .clone().detach() is the problem.

Resolved!
